src/test/java/com/artipie/test/TestContainer.java

```diff
@@ -106,4 +133,20 @@ public void close() {
         Logger.debug(this, "Command:\n%s", String.join(" ", command));
         return this.cntn.execInContainer(command);
     }
+
+    /**
+     * Log std result of the command execution.
+     * @param exec Result of the execution
+     * @param error Is it necessary to log stderr as error?
+     */
+    private void logStd(final Container.ExecResult exec, final boolean error) {
+        final String outputerr = String.format("\nSTDERR:\n%s", exec.getStderr());
+        if (error) {
+            Logger.info(this, String.format("\nSTDOUT:\n%s", exec.getStdout()));
+            Logger.error(this, outputerr);
+        } else {
+            Logger.warn(this, outputerr);
+        }
+    }
+
```

Comment:
@genryxy that looks like too complicated logic for output. I'd suggest to just print stdout and stderr after every command executed as `DEBUG` logging message and throw stderr content as exception if error code is not zero.